src/FieldType/Mapper/QueryFormMapper.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzPlatformQueryFieldType\FieldType\Mapper;
+
+use EzSystems\EzPlatformQueryFieldType\Form\Type\FieldType\QueryFieldType;
+use eZ\Publish\API\Repository\ContentTypeService;
+use EzSystems\RepositoryForms\Data\Content\FieldData;
+use EzSystems\RepositoryForms\Data\FieldDefinitionData;
+use EzSystems\RepositoryForms\FieldType\FieldDefinitionFormMapperInterface;
+use EzSystems\RepositoryForms\FieldType\FieldValueFormMapperInterface;
```

Comment:
`repository-forms` has been deprecated in favor of `ezplatform-content-forms`

Reply:
Well, I may have an issue... this package is compatible with 2.x and 3.x. How do I do this ?